app/src/main/java/com/blaizmiko/popcornapp/models/Movie/SpokenLanguage.java

```diff
@@ -0,0 +1,26 @@
+package com.blaizmiko.popcornapp.models.Movie;
+
+import com.google.gson.annotations.SerializedName;
+
+public class SpokenLanguage {
```

Comment:
It's a languages on what actors in the film spoken, besides main language of the film.